README.md

```diff
@@ -100,12 +99,10 @@ To configure the collection for your Tableau environment:
    10ay.online.tableau.com
    ```
 
-2. Select the authentication method you wish to use and declare the `CURRENT VALUE` for those fields in the environment file. A detailed description of available methods can be found in the [Authentication section of the API Reference](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_authentication.htm#sign_in). Authentication methods include Username & Password, [Personal Access Token](https://help.tableau.com/current/online/en-us/security_personal_access_tokens.htm) (PAT), and JWT (Connected Apps).
+2. Determine the authentication method you wish to use and declare the `CURRENT VALUE` for those fields in the environment file. You can find a list of the variables for your authentication method in [Tableau authentication credential and permissions](#tableau-authentication-credential-and-permissions). 
```

Comment:
I had most trouble with this step as I didn't know what fields to fill out. When I saw the table later, things started to click. That is why I added the second sentence. Happy to drop it if you think it would be more distracting than helping.

Reply:
Suggest:

2. Determine the authentication method you wish to use, and declare the `CURRENT VALUE` for the fields for that method in the environment file. You can find a list of variables for authentication methods that are available, and information about how users' credentials impact resources they have access to, in [Tableau authentication credential and permissions](#tableau-authentication-credential-and-permissions).